Fix 25936

divyajyothi5321 opened the pull request:
<!---
    Thank you for contributing to Magento.
    To help us process this pull request we recommend that you add the following information:
     - Summary of the pull request,
     - Issue(s) related to the changes made,
     - Manual testing scenarios
    Fields marked with (*) are required. Please don't remove the template.
-->

<!--- Please provide a general summary of the Pull Request in the Title above -->

### Description (*)
<!---
    Please provide a description of the changes proposed in the pull request.
    Letting us know what has changed and why it needed changing will help us validate this pull request.
-->

### Fixed Issues (if relevant)
<!---
    If relevant, please provide a list of fixed issues in the format magento/magento2#<issue_number>.
    There could be 1 or more issues linked here and it will help us find some more information about the reasoning behind this change.
-->
1. magento/magento2#25936: Regular Price Label Alignment Issues in Frontend

### Manual testing scenarios (*)
<!---
    Please provide a set of unambiguous steps to test the proposed code change.
    Giving us manual testing scenarios will help with the processing and validation process.
-->
1. Admin->Catalog->Products->Add Product
2. Add Price ex:$20
3. Click on Advanced Pricing and add regular price ex:$10
4. Save the product
5. Open product in frontend 
6. you can see the issue is fixed with this code as shown in the below screenshot.
![regularprice-expectedresult](https://user-images.githubusercontent.com/54176640/70617434-9ca39780-1c36-11ea-908a-8f3f0e2a059d.png)


### Questions or comments
<!---
	If relevant, here you can ask questions or provide comments on your pull request for the reviewer
	For example if you need assistance with writing tests or would like some feedback on one of your development ideas
-->

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

divyajyothi5321 committed:
Fixed #25901

divyajyothi5321 committed:
Added necessary indentations

divyajyothi5321 committed:
Merge pull request #1 from divyajyothi5321/#25901-fix-1

Fixed #25901

divyajyothi5321 committed:
Update _minicart.less

divyajyothi5321 committed:
added fix for #25936

divyajyothi5321 committed:
Update _minicart.less

divyajyothi5321 committed:
Update _minicart.less

ptylek commented:
@magento give me test instance

magento-engcom-team commented:
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

magento-engcom-team commented:
Hi @ptylek, here is your [new Magento instance](https://pr-25989.instances.magento-community.engineering).
Admin access: [https://pr-25989.instances.magento-community.engineering/admin](https://pr-25989.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

ptylek reviewed (changes requested):
@divyajyothi5321 Thanks for your contribution. Your solution breaks SKU into two lines, see the photo:
![image](https://user-images.githubusercontent.com/34317830/70624941-ab338400-1c21-11ea-9e34-828dc6882cd6.png)

divyajyothi5321 committed:
added required changes

When the length of SKU increases it will be falling down to the next line in mobile screens. Since there is no space available.

ptylek commented on app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less:
```diff
@@ -648,6 +648,20 @@
+//  Mobile @screen__xs
+//  _____________________________________________
+
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__xs) {
+    .product-info-main {
+        .product-info-price {
+            .price-box {
+                width: 1px;
+            }
+        }
+    }
+}
+
```
Doesn't make sense. Looks like this:
<img width="344" alt="Screenshot 2019-12-12 at 20 31 02" src="https://user-images.githubusercontent.com/34317830/70742698-63deed80-1d1e-11ea-8113-a30cf694aa12.png">

ptylek replied:
And I guess you want to achieve this:
![image](https://user-images.githubusercontent.com/34317830/70742744-86710680-1d1e-11ea-998d-e3259e90fc7b.png)

ptylek reviewed (changes requested):
Your changes break line on price box and sku. Screenshot was taken on ipad.
<img width="236" alt="Screenshot 2019-12-12 at 20 30 29" src="https://user-images.githubusercontent.com/34317830/70742846-c1733a00-1d1e-11ea-83c1-823638ab738f.png">

ptylek commented:
@divyajyothi5321, I am closing this PR now due to inactivity. Please reopen and update if you wish to continue. Thank you for collaboration
